frontend/src/modules/entitieslist/components/Entity.js

```diff
@@ -130,12 +144,19 @@ export default class Entity extends React.Component<Props> {
                     className='status fa'
                     onClick={this.toggleForBatchEditing}
                 />
+                {parameters.search || this.areFiltersApplied() ? (
```

Comment:
I'm not sure if we should make the icon only visible on hover or only for selected entities. Let's try with the selected entities first and see how it'll work (I'm not sure if there's a use case for seeing surrounding strings of strings that are not being edited).